Automattic/vip-cli
# Bump "@automattic/vip-search-replace": "^1.0.14",

## Description

Bump min version to pull in: 

https://github.com/Automattic/vip-search-replace/pull/17

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-search-replace.js` with various inputs
1. It should work as before (and should work on Apple M1 hardware)
